django/views/templates/default_urlconf.html

```diff
@@ -2,251 +2,244 @@
+        color: #092e20;
+        background-position-x: center;
+        background-repeat: no-repeat;
+        font-size: 40px;
+        font-size: 4rem;
+        display: block;
+        font-weight: 700;
+        margin-top: 16px;
+        overflow: hidden;
+        text-decoration: none;
+        text-indent: 100%;
+        width: 104px;
+        height: 36px;
```

Comment:
We should remove static `width`/`height` to allow for size adjustment when users use a different font size. I removed them and change `font-size` to `2rem`.